pkg/apis/policy/types.go

```diff
@@ -228,6 +228,8 @@ type PodSecurityPolicySpec struct {
 	// e.g. "foo.*" forbids "foo.bar", "foo.baz", etc.
 	// +optional
 	ForbiddenSysctls []string
+	// RunAsGroup is the strategy that will dictate the allowable RunAsGroup values that may be set.
+	RunAsGroup RunAsGroupStrategyOptions
```

Comment:
move to just after `RunAsUser`

Reply:
document this is alpha-level and requires the RunAsGroup feature gate to be enabled